mne_bids_pipeline/_config.py

```diff
@@ -1611,8 +1611,6 @@
 Must contain at least two elements. By default, 5 equally-spaced bins are
 created across the non-negative time range of the epochs.
 All specified time points must be contained in the epochs interval.
-If `None`, do not perform **time-frequency** analysis, and only run CSP on
-**frequency** data.
```

Comment:
This isn't what we actually do

Reply:
I haven't checked the code, but would there be a way to disable the creation of time bins? Should one pass an empty list here, then?

Reply:
It ended up being a bit complicated (`+203 −168`) but passing an empty list now works

Reply:
Oh wow!

Now I'm wondering if this is a feature we actually need (I should've asked that question before sending you off on this journey!!)

WDYT?

Reply:
Now that it's implemented I guess it's at least okay to have it